protocols.csv

```diff
@@ -1,15 +1,27 @@
-code,	size,	name
+code,	size,	name,	comment
 4,	32,	ip4
 6,	16,	tcp
 17,	16,	udp
 33,	16,	dccp
 41,	128,	ip6
+53,	V,	dns,	reserved for near-future use
+54,	V,	dns4,	reserved for near-future use
+55,	V,	dns6,	reserved for near-future use
 132,	16,	sctp
 301,	0,	udt
 302,	0,	utp
-421,	V,	ipfs
+400,	V,	unix
+420,	V,	p2p,	preferred over /ipfs
+421,	V,	ipfs,	equal to /p2p
+444,	96,	onion
 480,	0,	http
 443,	0,	https
-444,	96,	onion
-477, 0, ws
-275, 0, libp2p-webrtc-star
\ No newline at end of file
+477,	0,	ws
+478,	0,	wss
+275,	0,	libp2p-webrtc-star
+276,	0,	libp2p-webrtc-direct
+290,	V,	libp2p-relay
+
+16383,	V,	exp-dns,	experimental /dns
+16382,	V,	exp-dns4,	experimental /dns4
+16381,	V,	exp-dns6,	experimental /dns6
```

Comment:
> Do we need it to be experimental? Isn't it going to be more of an 'experimental implementation', than a 'experimental addr'?

It's probably experimental on both address semantics and implementation.

The practical reason is I'm not set on whether we should do only /dns, or only /dns[4,6], or both, and would like to implement both before settling on an option.

The process reason is I'm pretty confident moving fast with protocols prefixed with /libp2p- and protocols that don't have values (like /ws), because they're likely easy to change or abandon. I feel like the more essential protocols like /ip4 and /dns need more care.